website/docs/r/iam_trusted_profiles_claim_rule.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles_claim_rule"
+description: |-
+  Manages iam_trusted_profiles_claim_rule.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles_claim_rule
+
+Provides a resource for iam_trusted_profiles_claim_rule. This allows iam_trusted_profiles_claim_rule to be created, updated and deleted.
```

Comment:
Provides a resource for iam_trusted_profiles_claim_rule. This allows iam_trusted_profiles_claim_rule to be created, updated and deleted.

as and with the reference link.
Create, update, or delete an IAM trusted profiles claim rule resource. For more information, about IAM trusted profiles claim rule, see []().